doc/update034.md

```diff
@@ -6,3 +6,160 @@
+        __atomic_compare_exchange(p, oldval, newval);
+        return 0;
+        }
+EOL
+    if g++ atomic-builtin-test.cpp -I./db-4.8.30.NC/dbinc -DHAVE_ATOMIC_SUPPORT -DHAVE_ATOMIC_X86_GCC_ASSEMBLY -o atomic-builtin-test 2>/dev/null; then
+        echo "No changes to bdb source are needed ..."
+        rm atomic-builtin-test 2>/dev/null
+    else
+        echo "Updating atomic.h file ..."
+        sed -i 's/__atomic_compare_exchange/__atomic_compare_exchange_db/g' db-4.8.30.NC/dbinc/atomic.h
+    fi
+    cd db-4.8.30.NC/build_unix/
+    ../dist/configure -enable-cxx -disable-shared -with-pic -prefix=$Powerblockcoin_PREFIX
+    make install
+    cd $Powerblockcoin_ROOT
+}
+buildPowerblockcoin () {
+    git pull
+    ./autogen.sh
+    ./configure LDFLAGS="-L${Powerblockcoin_PREFIX}/lib/" CPPFLAGS="-I${Powerblockcoin_PREFIX}/include/" --with-gui=no --disable-tests --disable-bench --without-miniupnpc --enable-experimental-asm --enable-static --disable-shared --with-incompatible-bdb
+    make -j$(nproc)
+}
+berkeleydb
+buildPowerblockcoin
+echo "Done building Powerblockcoin!"
+sudo ln -sf /home/$USER/powerblockcoin-core/src/powerblockcoin-cli /usr/local/bin/powerblockcoin-cli
+sudo ln -sf /home/$USER/powerblockcoin-core/src/powerblockcoind /usr/local/bin/powerblockcoind
+```
+
+#### Step 3: Make the script executable and run it
+
+```bash
+chmod +x build.sh
+./build.sh
+```
+
+- Supply your `sudo` password when asked, so that the daemon and cli can be symlinked to your `/usr/local/bin` directory
+
+#### Step 4: Create Powerblockcoin data dir, powerblockcoin.conf file and restrict access to it
+
+```bash
+cd ~
+mkdir .powerblockcoin
+nano ~/.powerblockcoin/powerblockcoin.conf
+```
+
+Insert the following contents inside the powerblockcoin.conf file and save it. (change the rpcuser and rpcpassword values)
+
+```bash
+server=1
+daemon=1
+txindex=1
+rpcuser=user
+rpcpassword=password
+bind=127.0.0.1
+rpcbind=127.0.0.1
+rpcallowip=127.0.0.1
+```
+
+Restrict access to the powerblockcoin.conf file
+
+```bash
+chmod 600 ~/.powerblockcoin/powerblockcoin.conf
+```
+
+#### Step 5: Start the daemon, let it sync and monitor the debug.log
+
+- Run the following command to start the daemon
+
+```bash
+powerblockcoind &
+```
+
+- monitor the sync process by tailing the debug.log file in another terminal window
+
+```bash
+tail -f ~/.powerblockcoin/debug.log
+```
+
+#### Step 6: Import the privkey for Powerblockcoin corresponding to your 3p server
+
+Powerblockcoin's  address and wif format are the same as KMD. You can import your 3p KND node's wif into the Powerblockcoin daemon directly.
+
+Then import the privkey using the `importprivkey` command.
+
+Powerblockcoin's rpc calls are similar to BTC's after version `v0.18`. So familiar commands like `getinfo` don't work, and `validateaddress` doesn't output `ismine` info. To check if your address was imported, use `getaddressinfo` and check the `ismine` key in the response.
```

Comment:
```
$ powerblockcoin-cli getaddressinfo RNodeoneSNbxXEpoYXPfwJ7enKSscq7ZK3
error code: -32601
error message:
Method not found
```

Reply:
created a PR that corrects this https://github.com/KomodoPlatform/dPoW/pull/223